vip-core/src/main/java/fr/insalyon/creatis/vip/core/server/business/Server.java

```diff
@@ -245,6 +247,9 @@ private Server() {
             //Applicatoin importer
             applicationImporterFileRepository = config.getString(CoreConstants.APPLICATION_FILES_REPOSITORY, "/tmp/boutiques-cache");
             deleteFilesAfterUpload = config.getString(CoreConstants.APP_DELETE_FILES_AFTER_UPLOAD, "yes");
+            applicationImporterRootFolder = config.getString(CoreConstants.APP_IMPORTER_ROOT_FOLDER, "/biomed/user/c/creatis/vip/data/groups/Applications");
+            List<String> listApplicationImporterRequirements = new ArrayList<>();
+            applicationImporterRequirements = config.getList(CoreConstants.APP_REQUIREMENTS, listApplicationImporterRequirements);
```

Comment:
why not put `new ArrayList<>()` as `getList` 2nd parameter instead of using an useless variable ?

Reply:
Done also.